src/kani-compiler/rustc_codegen_kani/src/codegen/function.rs

```diff
@@ -184,8 +184,8 @@ impl<'tcx> GotocCtx<'tcx> {
         // ```
 
         let tupe = sig.inputs().last().unwrap();
-        let args: Vec<&TyS<'tcx>> = match tupe.kind() {
-            ty::Tuple(substs) => substs.iter().map(|s| s.expect_ty()).collect(),
+        let args = match tupe.kind() {
+            ty::Tuple(substs) => *substs,
             _ => unreachable!("a function's spread argument must be a tuple"),
         };
```

Comment:
You don't have to change this, obviously, but I see this pattern a couple of times and wonder if there's actually already a helper to do this in rustc...

Reply:
Let me check.

Reply:
I can only find these two occurrences, so I won't bother with a helper method at this point.